default.json

```diff
@@ -1,8 +1,7 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
-    "config:base",
```

Comment:
Is `best-practices` a super-set of `config:base`? And if not, what would be removed with this change?

Reply:
based on my understanding `config:base` is the one inherited by every other configuration. We can leave it in, in case. It should not change the computed final config.